README.adoc

```diff
@@ -12,3 +12,30 @@ The `xorg.conf` file should be placed in `/etc/X11/`.
 The `config.txt` file is the equivalent of a BIOS on the pi. It contains system configuration parameters. 
 This file is needed on the primary and secondary pi.
 The `config.txt` file should be placed in `/boot/`.
+
+= Instructions to run SetupScript
+
+Since the Script clones the Epsilon-Raspberry repo automatically, just download the script file itself to any preferred directory.
+
+Before running the setup script, ensure:
+
+* There is consistent, stable internet connection.
+* The pi and computer remain stationary for the duration of the script.
+The script may take several *hours* to run, so the computer may be on overnight.
+----
+sudo ./SetupScript
+----
+Add the following two lines to the `~/.bashrc' file:
+----
+export LD_LIBRARY_PATH=/usr/local/Qt-5.5.1/lib/:/usr/local/lib
+export PATH=/usr/local/Qt-5.5.1/bin:/usr/local/bin:/usr/bin:/bin:/usr/local/games:/usr/games:/usr/local
+----
+Save the changes and source the file to initialize the changes.
+----
+source ~/.bashrc
+----
+Check where it points, ideally located in `/usr/local/Qt-5.5.1/bin/qmake`
+----
+which qmake
+----
+Sample Applications can now be compiled and run.
```

Comment:
If sample applications refers to something in particular it should be styled somehow. Otherwise applications shouldn't be capitalized.